control-plane/tests/test_phase21_runtime_auth_validation.py

```
@@ -58,6 +62,64 @@ def _build_service(*, host: str = "127.0.0.1") -> AegisOpsControlPlaneService:
 
 
 class Phase21RuntimeAuthValidationTests(unittest.TestCase):
+    def test_operational_runtime_surfaces_are_extracted_into_dedicated_collaborators(
+        self,
+    ) -> None:
+        service = _build_service(host="0.0.0.0")
```

Ruff (0.15.9)

[error] Possible binding to all interfaces

(S104)